File: problem_solving_for_beginners/bitwise/python/disable_nth_bit.py
#!/usr/bin/python3
"""
    Disable nth bit of a given number using
    bitwise AND and LEFT SHIFT operators
"""
num, n = input("Enter Number and nth bit to disable: ").split()
num = int(num)
n = int(n) - 1

res  = num & ~(1 << n)

print(res)

File: problem_solving_for_beginners/bitwise/c-programming/disable_nth_bit.c
#include<stdio.h>
/***
 * main - Disable nth bit of a given number using bitwise
 *
 *
 * Return: prints ON otherwise OFF
 */
int main(void)
{
	int num, n, res;

    printf("Enter number and nth bit you want to disable: ");
	scanf("%d%d", &num,&n);

	//Write your code here
    res = num & ~(1 << (n - 1));

    printf("%d\n", res);

	return 0;
}